content/zh/docs/reference/glossary/kube-scheduler.md

```diff
@@ -28,10 +28,12 @@ tags:
 -->
 
 <!--
- Component on the master that watches newly created pods that have no node assigned, and selects a node for them to run on.
--->
+Control plane component that watches for newly created
+{{< glossary_tooltip term_id="pod" text="Pods" >}} with no assigned
+{{< glossary_tooltip term_id="node" text="node">}}, and selects a node for them
+to run on.-->
 
-主节点上的组件，该组件监视那些新创建的未指定运行节点的 Pod，并选择节点让 Pod 在上面运行。
+  控制平面组件，负责监视新创建的、未指定运行{{< glossary_tooltip term_id="节点（node）" text="node">}}的 {{< glossary_tooltip term_id="pod" text="Pods" >}}，选择节点让 Pod 在上面运行。
```

Comment:
you cannot change the term_id. You can change the text if you like. Ie

```suggestion
控制平面组件，负责监视新创建的、未指定运行{{< glossary_tooltip term_id="node" text="节点（node）">}}的
{{< glossary_tooltip term_id="pod" text="Pods" >}}，选择节点让 Pod 在上面运行。
```

Reply:
yes,I have updated the entry justly.